src/scss/_main.scss

```diff
@@ -1871,3 +1876,35 @@ select {
 .custom-select-container .custom-select-options:not(.hidden) {
   display: block;
 }
+
+.small.no-wrap {
+  display: flex;
+  align-items: center;
+
+  font-size: 0.8em;
+  white-space: nowrap;
+}
+
+.copy-text {
+  margin-right: 5px;
+  margin-left: 5px;
+
+  font-weight: bold;
```

Comment:
```suggestion

```

to match existing styles
<img width="446" alt="Screenshot 2025-01-22 at 10 41 27 AM" src="https://github.com/user-attachments/assets/4871efa3-c677-4bd3-9846-cec3559d6c84" />